src/list_ordered_multimap.rs

```diff
@@ -1669,6 +1670,22 @@ where
     }
 }
 
+impl<Key, Value, State> IntoIterator for ListOrderedMultimap<Key, Value, State>
+where
+    Key: Eq + Hash + Clone,
```

Comment:
Alphabetize traits

Reply:
Dunno what you mean here. There are three `impl`s for `IntoIterator`. I moved the owning `IntoIterator` up, so it reflects the order of `IntoIter`, `Iter`, `IterMut` as in the tests.

Reply:
I was referring to the trait bounds: Eq + Hash + Clone

Reply:
Ahh, that makes sense. Ok, I force pushed with trait bounds reordered, but kept the new `impl` order here. `IntoIter` comes now before `Iter` and `IterMut`. Since the code of all these three is structured similar, the diff looks quite complicated, but I basicaly cut all of `IntoIter` and pasted it before `Iter`.